### PR TITLE
[WIP] prism overview article

### PR DESCRIPTION
**Pull request by pytlesk4**

### CODE REVIEW

File: articles/prism/overview.md [hunk 1]
```diff
@@ -1,62 +1,97 @@
...
+* Act as a transformation layer, manipulating incoming requests and outgoing responses
+* Act as a validation layer, validating incoming requests and outgoing responses
+* Contract test your APIs, given an OAS(Swagger 2) file
+* Log all or a subset of traffic to configurable locations
+* Extend existing APIs with new endpoints or capabilites
+* Act as a system-wide proxy, blocking traffic to particular websites or endpoints
+
+## Conduct vs Serve
+
+Conduct and Serve are important concepts to understand when using Prism. Conduct is an isolated scenario run. You tell Prism what scenarios you want to run, give it an environment to run against, and Prism will generate a report of the run. Serving, on the other hand, is a long running instance of Prism that applies scenarios to HTTP traffic.
```
Comment (rowa97) [inline articles/prism/overview.md]: "Conduct is an isolated scenario run" is a bit awkward, what about "Conduct runs an isolated scenario"

File: articles/prism/overview.md [hunk 2]
```diff
@@ -1,62 +1,97 @@
...
+
+* Act as a mock server, routing incoming requests to example repsonses, or dynamically generating examples on the fly
+* Act as a transformation layer, manipulating incoming requests and outgoing responses
+* Act as a validation layer, validating incoming requests and outgoing responses
+* Contract test your APIs, given an OAS(Swagger 2) file
+* Log all or a subset of traffic to configurable locations
+* Extend existing APIs with new endpoints or capabilites
+* Act as a system-wide proxy, blocking traffic to particular websites or endpoints
+
+## Conduct vs Serve
```
Comment (rowa97) [inline articles/prism/overview.md]: conduct and serve should also be introduced in the beginning paragraphs or in the features list

Comment (pytlesk4) [reply articles/prism/overview.md]: Good call!

File: articles/prism/overview.md (hunk 1 above)
Comment (pytlesk4) [reply articles/prism/overview.md]: Going to push changes today!

**Comment by pytlesk4** [comment]
Let's get this up today! Made the requested changes

**Pull request merged**